ramlfications/loader.py

```diff
@@ -50,9 +50,14 @@ def _parse_json(self, jsonfile, base_path):
             base_path = base_path + "/"
         base_path = "file://" + base_path
 
-        with open(jsonfile, "r") as f:
-            schema = jsonref.load(f, base_uri=base_path, jsonschema=True)
-        return schema
+        try:
+            with open(jsonfile, "r") as f:
+                schema = jsonref.load(f, base_uri=base_path, jsonschema=True)
+        except ValueError as err:
+            msg = "ERROR: failed to parse JSON file: {0} ({1})"
+            raise LoadRAMLError(msg.format(jsonfile, err))
```

Comment:
Yep! There are a couple of examples of asserting that an error was raised in `tests/test_loader.py`. I'm assuming it could be similar to the `test_yaml_parser_error` test.